package/chalicelib/awsAuditor.py

```diff
@@ -0,0 +1,32 @@
+from package.chalicelib.reportGenerator import ReportGenerator
```

Comment:
Probably just pull down what we currently have here and make the necessary changes to deploy it.